tests/Unit/Evolution/Systems/GeneralizedHarmonic/Test_Tags.cpp

```diff
@@ -0,0 +1,69 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "tests/Unit/TestingFramework.hpp"
+
+#include <string>
+
+#include "DataStructures/DataBox/DataBoxTag.hpp"
+#include "Evolution/Systems/GeneralizedHarmonic/Tags.hpp"
+
+namespace {
+struct ArbitraryFrame;
+}  // namespace
+
+template <size_t D, typename F>
```

Comment:
Don’t use single-letter names please, but name them `Dim`, `Frame` and `DataType`. Single letters are not only less informative, they also make refactoring harder as you will notice when making this change ;)

Reply:
changed